### ISSUE TITLE
games page missing reference to box2d

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#791]
```
Please include box2d in /games tools
```

Original issue reported on code.google.com by `sethladd@google.com` on 1 Feb 2012 at 2:50

**Comment by GoogleCodeExporter** [comment]
```
Lilli, can you add this to the tools section?
```

Original comment by `ericbide...@html5rocks.com` on 1 Feb 2012 at 4:13
- Changed state: **Assigned**
- Added labels: **Milestone-Q12012**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 3 Feb 2012 at 11:25

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 55e8ac19a57c.
```

Original comment by `ericbide...@html5rocks.com` on 3 Feb 2012 at 11:26
- Changed state: **Fixed**
